CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Add `UploadedAttachmentPostProcessor` in `ChatClientConfig` to allow changing custom attachment payloads after an attachment has been uploaded [#2457](https://github.com/GetStream/stream-chat-swift/pull/2457)
 - Add `AnyAttachmentPayload(localFileURL:customPayload:)` initializer to allow creating custom attachments without a remote URL [#2457](https://github.com/GetStream/stream-chat-swift/pull/2457)
 - Add skip push support when sending a message [#2486](https://github.com/GetStream/stream-chat-swift/pull/2486)
+- Add `isChannelLocalFilteringEnabled` in `ChatClientConfig` to allow changing whether the Channels in ChannelList will be locally filtered [#2488](https://github.com/GetStream/stream-chat-swift/pull/2488)
```

Comment:
I think that `isChannelAutomaticFilteringEnabled` is a bit more suggestive for customers 🤔 Since "Local" is kinda an implementation detail and doesn't actually express what the goal of it is

Reply:
@ipavlidakis, also don't forget these docs: https://getstream.io/chat/docs/sdk/ios/client/controllers/channels/#2-create-a-controller, they might require an update.

Reply:
Yes, they should touch on the points described here: https://github.com/GetStream/stream-chat-swift/pull/2488#pullrequestreview-1299334895